src/activities/AnshikaMittal/AnshikaMittal.qml

```diff
@@ -0,0 +1,503 @@
+/* GCompris - AnshikaMittal.qml
+ *
+ * Copyright (C) 2016 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
+ *   YOUR NAME <YOUR EMAIL> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+
+import GCompris 1.0
+import QtQuick.Controls 1.0
+import "../../core"
+import "AnshikaMittal.js" as Activity
+
+ActivityBase {
+    id: activity
+    property color buttonColor: "lightblue"
+    property color onHoverColor: "gold"
+    property color borderColor: "white"
+
+
+
+    onStart: focus = true
+    onStop: {}
+
+
+    pageComponent: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: "#ABCDEF"
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        GCText {
+            //  anchors.centerIn: parent
+            text: "Anshika Mittal's activity"
+            x:870; y:670;
+            fontSize: mediumSize
+        }
+
+        DialogHelp {
+            id: dialogHelp
+            onClose: home()
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level }
+            onHelpClicked: {
+                displayDialog(dialogHelp)
+            }
+            onPreviousLevelClicked: Activity.previousLevel()
+            onNextLevelClicked: Activity.nextLevel()
+            onHomeClicked: activity.home()
+        }
+
+        Bonus {
+            id: bonus
+            Component.onCompleted: win.connect(Activity.nextLevel)
+        }
+
+        Image{
+            id: img1
+
+               width : 200;
+               height: 250;
+                x: 950
+                y:350
+                   source: "qrc:/gcompris/src/activities/AnshikaMittal/resource/blocks.svg"
+                   anchors.fill: parent.BottomLeft
```

Comment:
anchors.fill should be use with an Item.
If you want it on bottom left, you should do 
anchors.bottom: parent.bottom
anchors.right: parent.right